examples/event-webhooks-local-flask/local_app/lib/postgresql.py

```diff
@@ -0,0 +1,44 @@
+import os
+from pathlib import Path
+
+import psycopg2
+
+from local_app.lib.logger import get_logger
+
+logger = get_logger()
+
+
+def write_data(entity_name: str, benchling_api_id: str, field_value: str | None) -> str:
```

Comment:
@edwin-benchling This is the third and final file that is new and merits closer review.

Although I don't wanna focus too hard on teaching users how to connect to PostgreSQL in Python and intentionally kept this light.